src/store/RootStore/AllRecipesStore/AllRecipesStore.ts

```diff
@@ -0,0 +1,71 @@
+import { getSixRecipes } from '@services/recipes';
+import { recipe } from '@store/models';
+import {
+  makeObservable,
+  observable,
+  computed,
+  action,
+  runInAction,
+} from 'mobx';
+
+import { Meta } from './types';
+
+type PrivateFields = '_recipes' | '_meta' | '_maxRecipes';
+
+export default class AllRecipesStore {
+  private _recipes: recipe[] = [];
+  private _meta: Meta = Meta.initial;
+  private _rootStore;
+  private _maxRecipes: number = 0;
+
+  constructor(rootStore: any) {
+    // Какая-то сомнительная история с этим типом. Непонятно, как его правильно написать
+    this._rootStore = rootStore;
+    makeObservable<AllRecipesStore, PrivateFields>(this, {
+      _recipes: observable.ref,
+      _meta: observable.ref,
+      _maxRecipes: observable.ref,
```

Comment:
аналогично числам, строки можно помечать как просто observable и пометка их как observable.ref не даст прироста производительности, лишь запутает читателя
```suggestion
      _recipes: observable.ref,
      _meta: observable,
      _maxRecipes: observable,
```